merchant-sdk/sdk/src/main/java/net/gini/android/merchant/sdk/GiniMerchant.kt

```diff
@@ -125,7 +125,7 @@ class GiniMerchant(
      * successful payment requests or which screen is being displayed.
      */
 
-    private val _eventsFlow: MutableSharedFlow<MerchantSDKEvents> = MutableSharedFlow(extraBufferCapacity = 1)
+    private val _eventsFlow: MutableSharedFlow<MerchantSDKEvents> = MutableSharedFlow(extraBufferCapacity = 1,replay = 1)
```

Comment:
Added `replay` for this `sharedFlow` for the following use case:
- `ShareSheet` shown -> tap on an application -> tap event emitted -> the tap is received in the `viewModel` -> would be received in `onResume` of the fragment.

Since it's a `sharedFlow`, this value will be lost before `onResume` would be called. Adding `replay` will emit the last value when receivers register.